tests/e2e/framework/common.go

```diff
@@ -95,3 +156,282 @@ func (f *Framework) waitForScanCleanup() error {
 	}
 	return nil
 }
+
+func (f *Framework) addFrameworks() error {
```

Comment:
Note for reviewers.

These functions are pretty close to what existed in the helpers.go file, but the method signatures are different since we don't need to pass in additional things like namespaces or the framework when we have all that data associated to the framework already.